packages/core/src/generatePolicy.js

```diff
@@ -74,6 +74,9 @@ function createModuleInspector(opts) {
     moduleRecord,
     { isBuiltin, includeDebugInfo = false }
   ) {
+    if (moduleRecord === undefined) {
```

Comment:
This is redundant for the bugfix in question but it's the correct thing to do here anyway.